test/spec/modules/userId_spec.js

```diff
@@ -667,6 +667,7 @@ describe('User ID', function () {
 
       const ids = {
         'uid2': { id: 'uid2_value_from_mockId3Module' },
+        'uid1': { id: 'uid1_value_from_mockId3Module' },
```

Comment:
I think you should add `uid1: ['mockId1Module', 'mockId3Module']` in line 657 to have the priorities explicit and this line you then can test if the value from the mockId1Module has won - if I understand the intention of the test correctly. 

Maybe it would also be worth putting it into a separate test case?